trappy/systrace.py

```diff
@@ -88,9 +88,14 @@ def generate_data_dict(self, data_str):
 
         match = SYSTRACE_EVENT.match(data_str)
         if match:
-            data_dict = { 'event': match.group('event'),
-                          'pid'  : match.group('pid'),
-                          'func' : match.group('func'),
-                          'data' : match.group('data') }
+            pid = match.group('pid')
```

Comment:
Well, I guess these tree lines can be removed now that you added the check inline, isn't it?